lib/effects/effects_handler.dart

```diff
@@ -0,0 +1,48 @@
+import '../components/component.dart';
+import 'effects.dart';
+
+export './move_effect.dart';
+export './rotate_effect.dart';
+export './scale_effect.dart';
+export './sequence_effect.dart';
+
+class EffectsHandler {
+  /// The effects that should run on the component
+  final List<ComponentEffect> _effects = [];
+
+  EffectsHandler();
```

Comment:
is this necessary? if there are no other constructors dart generates the default constructor for you I think

Reply:
True, there was something in there at first and I just forgot to remove it. Updated now.